5.5/README.md

```diff
@@ -40,6 +40,8 @@ The following environment variables influence the MySQL configuration file. They
 |  `MYSQL_INNODB_LOG_BUFFER_SIZE` | The size of the buffer that InnoDB uses to write to the log files on disk | 8M (or 15% of available memory)
 |  `MYSQL_DEFAULTS_FILE`          | Point to an alternative configuration file                        |  /etc/my.cnf
 |  `MYSQL_BINLOG_FORMAT`          | Set sets the binlog format, supported values are `row` and `statement` | statement
+|  `MYSQL_LOG_QUERIES`            | Turns on logging of queries | 0
```

Comment:
we've generally tried to use true/false (and name it MYSQL_LOG_QUERIES_ENABLED), is "0" and "1" the values mysql requires for its actual configuration to enable this?